var/spack/repos/builtin/packages/mvapich2-gdr/package.py

```diff
@@ -124,12 +153,7 @@ def process_manager_options(self):
                 '--with-pm=hydra',
                 '--with-pbs=/opt/pbs'
             ])
-            if '~mcast' in spec:
-                opts.append('--disable-mcast')
-            if '+core_direct' in spec:
-                opts.append('--with-core-direct')
-            if '+openacc' in spec:
-                opts.append('--enable-openacc')
```

Comment:
Can you comment why these conditional branches have been removed? Are the variants involved used anywhere else or should they be removed too?

Reply:
Of course. If you look at line 123 in the update file, those same conditional branches were already present. I removed these because they were redundant.